src/accounts.js

```diff
@@ -90,16 +90,16 @@ function randomEosBase32() {
   return eosBase32(Math.random().toString(BASE).substr(2));
 }
 
-function generateAccountNameString() {
-  return (timestampEosBase32() + randomEosBase32()).substr(0, 12);
+function generateAccountNameString(prefix = '') {
+  return (prefix + timestampEosBase32() + randomEosBase32()).substr(0, 12);
```

Comment:
What's the time resolution here? is it like .1 second or what? Would like to know how likely a collision is if we create accounts quickly

Reply:
The resolution, without the prefix, is 1 unique, per millisecond, with an extra 3 characters of randomness. So, up to 1000 uniques per millisecond (best-case).

Adding an "ore" prefix will remove the randomness, and put the resolution at 1 unique per millisecond.